CONTRIBUTING/README.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+We aspire to create a welcoming environment for collaboration on this project
+and ask that all contributors do the same. For more details, see our [code of
+conduct](/CODE-OF-CONDUCT.md).
+
+This document covers contributions to this git repository.
+
+## Open source
+
+While this repository does not contain open source code, we manage content
+contributions following open source practice, as detailed below.
+
+All contributions to this project will be released under open source license as
+described in [LICENSE.md](/LICENSE.md). By submitting a pull request (PR),
+you are agreeing to release the PR contents under this license.
+
+## Communication
+
+Anyone interested in contributing should join the mailing list and other
+[communication channels](https://github.com/cncf/wg-env-sustainability#meetings)
+
+We strongly encourage and support all our members to participate in anyway
+they can. Not everyone can participate in the regularly scheduled live meetings,
+so we strive to make our processes friendly for people to be active contributors
+through asynchronous communication and contributions to our documentation
+in this repository.
+
+## Github pull requests and issues
+
+If you are new to the group, [reviewing pull requests](pull-request-review.md)
+and commenting on issues is a great way to get involved!
+
+When creating or reviewing pull requests, please refer to the
+[writing style guide](writing-style.md) to help maintain consistency across
+all of our documents.
+
+## Discussions
+
+You are welcomed to open discussions to the community and to bring in your ideas. Reach out to us through our Slack channel, mailing list or GitHub Discussion.
+
+## Priviliged Access and Rights
+
+If you need additional access rights for the WG GitHub Repository and/or the [Project Board](https://github.com/orgs/cncf/projects/10), please open a [simple issue](https://github.com/cncf/wg-env-sustainability/issues/new) with the request.
```

Comment:
```suggestion
If you need additional access rights for the TAG GitHub Repository and/or the [Project Board](https://github.com/orgs/cncf/projects/10), please open a [simple issue](https://github.com/cncf/tag-env-sustainability/issues/new) with the request.
```

I will add a note to the agenda to check the project board at the end of a meeting (if there is time) right now the project board is not very well maintained tbh.